docs/controller-skills/milkrun.md

```diff
@@ -0,0 +1,221 @@
+    <span class="hotline">**SFL** -> **GUN**</span>: "QFA494 via ODALE, 34R, Roger"  
+
+In order to reduce this coordination, GUN/BIK and SFL/SAS may organise blanket coordination to allow this rerouting at the discretion of the GUN/BIK controller.
+
+## Workload Management
+### Splitting
+Never ever ever ever ever (ever) deny an offer to split up your sector! It could be someone offering to take:  
+- **BLA** from **ELW**  
+- **WOL** from **BIK**  
+- **SY_DEP** from **SY_APP**  
+- **SY-E_GND** from **SY_GND**  
+The event gets busy every week, and denying offers to split takes away from the potential pool of controllers to help the event run smoothly. It is better to have a bit of a quieter half an hour, then be ready for the busy rush, than it is to have a decent amount of traffic for half an hour, and get completely snowed under after that.
+
+#### Prioritisation
+Some sectors generally require splitting more than others. For example, the Melbourne TMA can usually operate without the need for a Departures controller for the whole event (although if there's no room for an S3 anywhere else, it's still worth having), whilst a second/third controller in the Sydney TMA is always extremely beneficial. Enroute splits such as WOL and BLA are also extremely beneficial to balancing the workload of the event.
+
+It can also be beneficial to redistribute the ratings across the event. For example, if Sydney TMA has two or three C1+ controllers on, whilst ELW is covering an offline BIK, it would be prudent for one of the Sydney TMA controllers to move up and cover BIK.
+
+#### Projecting
+Project **ahead of time** when your sector will be busy, by looking at things like:  
+- ADEP Strip List Length  
+- Announced Strip List Length  
+- Number of Blue Announced tracks on screen  
+- Number of Blue Announced tracks when zoomed out  
+
+Use this information to know that you will be busy soon, and put out a request for a split in the [atc-coordination](https://discord.com/channels/343999482737721354/613736441717194755){target=new} discord channel. If you start to encounter things like:  
+- Aircraft stepping on each other  
+- Multiple aircraft having to ask for further climb/descent  
+- Aircraft not listening to your calls  
+It may already be too late! And to make matters worse, if you wait too long to ask, there might not be any controllers available to split the sector.
+
+### Coordination
+Regular Coordination with your peers is critical to the efficient operation of a Milk Run. Not only from a control perspective (eg, asking for an early handoff, requesting an amended route, suggested heading for separation between TMA/ENR, etc), but also from a more long-term event management perspective. If things are getting too busy in your airspace, or are about to get very busy, you may elect to:  
+- Ask the ADC controller for additional departure spacing  
+- Temporarily Stop departures  
+- Ask the adjacent Enroute controller for additional spacing
```

Comment:
Could we add an example of TMA telling ENR that there's been a go around, so they need an extra gap in the sequence